airflow/contrib/example_dags/example_kubernetes_executor_config.py

```diff
@@ -45,7 +45,8 @@ def test_volume_mount():
             foo.write('Hello')
 
         return_code = os.system("cat /foo/volume_mount_test.txt")
-        assert return_code == 0
+        if not return_code:
```

Comment:
I prefer the previous assertions because it expresses intentions better.

> Explicit is better than implicit.

https://www.python.org/dev/peps/pep-0020/

Reply:
Changed it (and added more info in the exception).